powerline/config_files/themes/powerline_unicode7.json

```diff
@@ -143,6 +143,15 @@
 			"args": {
 				"text": "🖫⃥"
 			}
+		},
+
+		"powerline.segments.i3wm.scratchpad": {
+			"args": {
+				"icons": {
+					"fresh": "O",
+					"changed": "X"
```

Comment:
How is UTF-7 related? To get a unicode character given its code there is `<C-v>uXXXX`.